tsconfig.json

```diff
@@ -29,7 +29,6 @@
     "playwright.config.js",
     "injected",
     "injected/src/globals.d.ts",
-    "injected/src/features/duckplayer/duckplayer-settings.ts",
```

Comment:
Reminder to resolve this and the removed duckplayer-settings.d.ts before merging

Reply:
We don't use it, I can trim it out of the PR and make a separate one if you prefer?

Reply:
Ah, I thought maybe this wasn't intentional. But if it's intentional then by all means I'm fine with it here.